README.md

```diff
@@ -14,6 +14,14 @@ With Leiningen/Boot:
 [clj-meowallet "0.0.1"]
 ```
 
+#### Tests
+
+* `lein test` - runs the test suite *
+* `script/autotest` -listen for file changes and is always running tests
+
+* To Run the integration test you should provide a valid Meo Wallet API KEY with access to mb references api.
+  `TOKEN:"wejkqwhekjqwkje" lein test`
```

Comment:
How about a more specific name to the `TOKEN`? Is this a json token? auth token? Something like `MEO_WALLET_TOKEN`.